makefiles/Makefile.java.mk

```diff
@@ -300,6 +300,7 @@ $(JAVA_OR_TOOLS_LIBS): \
  $(GEN_PATH)$Scom$Sgoogle$Sortools$Sgraph$S*.java \
  $(GEN_PATH)$Scom$Sgoogle$Sortools$Slinearsolver$S*.java
 	"$(JAR_BIN)" cvf $(LIB_DIR)$Scom.google.ortools.jar -C $(CLASS_DIR) com$Sgoogle$Sortools$S
+	"$(JAR_BIN)" cvf $(LIB_DIR)$Scom.google.ortools-sources.jar -C $(GEN_DIR) com
```

Comment:
Not sure this contains all the sources. I guess you need to include generated sources, they will be on genS$com

Reply:
Hi @lperron , sorry for my very late reply! I was very busy last year from summer on :(.
The command actually executed is: `"/usr/lib/jvm/java-1.8.0-openjdk/bin/jar" cvf lib/com.google.ortools-sources.jar -C ortools/gen com`. So if I understand your comment correctly, this should include the generated sources. Is this correct or do I miss something?